tests/libraries/nested/data/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "pushd $tmp"
+        rlRun "rlImport tests/nested/file"
```

Comment:
Just want to make sure: Shall the `rlImport` command be given the repository name as well? Shouldn't it be just `rlImport nested/file`. @sopos, any thoughts on this?

Reply:
Yeah, repository name was always part of the rlImport path. As discussed privately.

Reply:
Library naming confirmed to be ok on the hacking session as well.